execution-plan/src/constants.rs

```diff
@@ -0,0 +1,26 @@
+use crate::Memory;
+use kittycad_execution_plan_traits::{Address, NumericPrimitive, Primitive};
+
+// Rust thinks this is dead-code but it will absolutely be used by consumers and
+// it is in fact used in tests.
+
+#[allow(dead_code)]
+pub const E: Primitive = Primitive::NumericValue(NumericPrimitive::Float(std::f64::consts::E));
+
+#[allow(dead_code)]
+pub const PI: Primitive = Primitive::NumericValue(NumericPrimitive::Float(std::f64::consts::PI));
+
+#[allow(dead_code)]
+pub struct Constant;
+
+impl Constant {
+    #[allow(dead_code)]
+    pub fn value(mem: &mut Memory, value: Primitive) -> Address {
```

Comment:
Any reason this invoked as `Constant::value` rather than being a normal function like `constant_value`?

Reply:
No reason - I thought you'd actually prefer what I thought was more idiomatic here.